Networking/Networking/Remote/ReceiptRemote.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+public final class ReceiptRemote: Remote {
+    /// Retrieves a `Receipt`for a given `siteID`and `orderID`
+    ///
+    /// - Parameters:
+    ///    - siteID: site ID which contains the receipt
+    ///    - orderID: ID of the order that the receipt is associated to
+    ///    - forceRegenerate: whether a new receipt is generated. Defaults to `true`
+    ///    - completion: callback with the expected `Receipt` object, or an error.
+    ///
+    public func retrieveReceipt(siteID: Int64,
+                                orderID: Int64,
+                                forceRegenerate: Bool = true,
```

Comment:
For the time being we intend to always force receipt regeneration when calling the endpoint, this might change later  (specially if we end calling it upon view load or refresh control), but for the moment we'll pass `true` as default parameter.